types/better-sqlite3/index.d.ts

```diff
@@ -5,8 +5,9 @@
 //                 Santiago Aguilar <https://github.com/sant123>
 //                 Alessandro Vergani <https://github.com/loghorn>
 //                 Andrew Kaiser <https://github.com/andykais>
+//                 Mark Stewart <https://github.com/mrkstwrt>
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
-// TypeScript Version: 3.0
+// TypeScript Version: 3.1
```

Comment:
What change mandated increasing the minimum TS version?

Reply:
My mistake, I thought this was the type package version, will change